lib/sales_analyst.rb

```diff
@@ -52,11 +52,6 @@ def average_average_price_per_merchant
     (total / merchants.all.count).round(2)
   end
 
-  def calculate_stdev(mean)
-    variance = mean.sum / (mean.count - 1)
-    Math.sqrt(variance).round(2)
-  end
-
```

Comment:
where did they go?

Reply:
They went to the merchant_analyst module